test/ghcjs-overlay/default.nix

```diff
@@ -6,14 +6,13 @@ let
   project = cabalProject' {
     src = testSrc "ghcjs-overlay";
     inherit compiler-nix-name evalPackages;
-    cabalProjectLocal = ''
-      allow-newer: *:*
-    '';
-    # Alternative to the --sha256 comment in cabal.project
-    # sha256map = {
-    #  "https://raw.githubusercontent.com/input-output-hk/hackage-overlay-ghcjs/bfc363b9f879c360e0a0460ec0c18ec87222ec32" =
-    #    "sha256-g9xGgJqYmiczjxjQ5JOiK5KUUps+9+nlNGI/0SpSOpg=";
-    # };
+    cabalProjectLocal = builtins.readFile ../cabal.project.local + (if stdenv.hostPlatform.isGhcjs
```

Comment:
Any reason not to put these into the actual cabal.project.local?

Reply:
This is a work around for double-conversion being out of date in ghcjs-overlay.  Not sure how easy that will be to fix (depends if there are new C functions added).  I did not want to include the work around in the shared `cabal.project.local` as it might not be needed soon.  I have moved it into this test's cabal.project though.